### PR TITLE
[TEVA-2031] Job Application Form Preview

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2031

## Link to Designs

https://www.figma.com/file/nXpc1rHULqHsAfhRXYUgsb/Sprint-75?node-id=1441%3A394

## Product sign off
- [x] Lovely jubbly!

## Changes in this PR:

- Added notification banner to applying for job step
- Added static view to display the job application preview

### Notification banner

Screenshot 2021-04-21 at 11 45 30

## Comments 

- I believe you should be able to see the page by clicking on the review app and then going to /pages/job-application-preview

### CODE REVIEW

File: app/views/pages/job-application-preview.html.slim [hunk 1]
```diff
@@ -0,0 +1,211 @@
+- content_for :page_title_prefix do
```
Comment (cpjmcquillan) [inline app/views/pages/job-application-preview.html.slim:1]: can we do it on one line instead like `- content_for :page_title_prefix, "Job Application Preview"`

File: app/views/pages/job-application-preview.html.slim [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  | Job Application Preview
+
+.govuk-main-wrapper
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      h2.govuk-heading-l id="heading" Teaching Vacancies application form
```
Comment (cesidio) [inline app/views/pages/job-application-preview.html.slim:7]: You should use default slim syntax for adding ids:
`h2#heading.govuk-heading-l Teaching Vacancies application form`
But why do you give it an id with name `heading`? it seems more a class for me

Comment (josephhull676) [reply app/views/pages/job-application-preview.html.slim:7]: Sure, I'll do that now. I should've remember this as we discussed it recently!

The id 'heading' was added for the 'back to top' anchor links.

File: app/views/publishers/vacancies/build/applying_for_the_job.html.slim [hunk 3]
```diff
@@ -13,6 +13,10 @@
 
         h2.govuk-heading-l = t("jobs.applying_for_the_job")
 
+        = govuk_notification_banner title: t("banners.important") do |banner|
+          - banner.add_heading text: t(".notification.heading")
+          p.govuk-body = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
```
Comment (cpjmcquillan) [inline app/views/publishers/vacancies/build/applying_for_the_job.html.slim:18]: You don't need the paragraph element here

File: app/views/pages/job-application-preview.html.slim [hunk 4]
```diff
@@ -0,0 +1,210 @@
+- content_for :page_title_prefix, "Job Application Preview"
+
+.govuk-main-wrapper
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      h2.govuk-heading-l#heading Teaching Vacancies application form
+
+      p.govuk-body-m
```
Comment (cpjmcquillan) [inline app/views/pages/job-application-preview.html.slim:8]: I think `.govuk-body` is `.govuk-body-m` by default

**Review by cpjmcquillan — APPROVED** [review]
Just a couple of comments

File: app/views/publishers/vacancies/build/applying_for_the_job.html.slim [hunk 5]
```diff
@@ -13,6 +13,10 @@
 
         h2.govuk-heading-l = t("jobs.applying_for_the_job")
 
+        = govuk_notification_banner title: t("banners.important") do |banner|
+          - banner.add_heading text: t(".notification.heading")
+          .govuk-body = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
```
Comment (cpjmcquillan) [inline app/views/publishers/vacancies/build/applying_for_the_job.html.slim:18]: You don't need any element at all here, if you do then the banner will have the extra margin

**Pull request merged**